src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+
+const compareAsStrings = (a, b) => {
+  if (String(a) > String(b)) {
+    return 1;
+  }
+
+  if (String(a) < String(b)) {
+    return -1;
+  }
+
+  return 0;
+};
```

Comment:
```suggestion
const compareAsStrings = (a, b) => String(a) > String(b);
```
A lot of useless code.

Reply:
Better encapsulate this function with sort2 method.